pkg/policy/l4.go

```diff
@@ -88,7 +88,8 @@ type L4Filter struct {
 	// Endpoints limits the labels for allowing traffic (to / from). If
 	// Endpoints is empty, then it selects all endpoints.
 	Endpoints []api.EndpointSelector `json:"-"`
-	// L7Parser specifies the L7 protocol parser (optional)
+	// L7Parser specifies the L7 protocol parser (optional). If specified as
+	// an empty string, then means that no L7 forwarding is performed to the proxy.
```

Comment:
L7 forwarding is performed to the proxy -> L7 proxy redirect is performed
(we've been using the term "redirect" quite consistently)